docker/Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN go mod download
 COPY . ./
 
 ARG GOFLAGS
-RUN go install github.com/stellar/go/services/galexie
```

Comment:
this might have inadertently gone right to install module from remote gh url instead of build from local code?